doc/message-registry.md

```diff
@@ -0,0 +1,253 @@
+# Error index {#error-index}
+
+<!--
+NOTE: This file has been generated automatically by “update-message-registry.py”.
+      Do not edit manually!
+-->
+
+This page lists the warnings and errors generated by xkbcommon.
+There are currently 15 entries.
+
+@todo The documentation of the log messages is a work in progress.
+
+## Index
+
+| Code      | Identifier                   | Description | Type |
+| --------- | ---------------------------- | ----------- | ---- |
+| [XKB-107] | `unrecognized-keysym` | Warn on unrecognized keysyms. | Warning |
+| [XKB-183] | `cannot-infer-key-type` | Warn if no key type can be inferred. | Warning |
+| [XKB-286] | `undefined-key-type` | Warn if using an undefined key type. | Warning |
+| [XKB-305] | `non-base-group-name` | Warn if a group name was defined for group other than the first one. | Warning |
+| [XKB-461] | `conflicting-key-symbol` | Warn if there are conflicting keysyms while merging keys. | Warning |
+| [XKB-489] | `numeric-keysym` | Warn on numeric keysym (other than 0-9). | Warning |
+| [XKB-516] | `extra-symbols-ignored` | <span class="todo">TODO:</span> add description | Warning |
+| [XKB-700] | `multiple-groups-at-once` | Warn if a key defines multiple groups at once. | Warning |
+| [XKB-769] | `invalid-syntax` | The syntax is invalid and the file cannot be parsed. | Error |
+| [XKB-770] | `undefined-keycode` | <span class="todo">TODO:</span> add description | Warning |
+| [XKB-800] | `conflicting-modmap` | Warn if there are conflicting modmap definitions. | Warning |
+| [XKB-883] | `conflicting-key-action` | Warn if there are conflicting actions while merging keys. | Warning |
+| [XKB-893] | `conflicting-key-type` | Warn if there are conflicting key types while merging groups. | Warning |
+| [XKB-935] | `conflicting-key-fields` | Warn if there are conflicting fields while merging keys. | Warning |
+| [XKB-965] | `unresolved-keymap-symbol` | Warn if using a symbol not defined in the keymap. | Warning |
+
+## Details
+
+### XKB-107 – Unrecognized keysym {#XKB-107}
+
+<dl>
+  <dt>Since</dt><dd>1.0.0</dd>
+  <dt>Type</dt><dd>Warning</dd>
+  <dt>Summary</dt><dd>Warn on unrecognized keysyms.</dd>
+</dl>
+
+xkbcommon replaces keysyms it does not recognize by the keysym `NoSymbol`.
+
+You may find the list of supported keysyms in
+`include/xkbcommon/xkbcommon-keysyms.h`.
+
+
+#### Examples
+
+<details>
+  <summary>Unrecognized keysym “`coma`”</summary>
+
+__Error message:__
```

Comment:
I'm feeling a bit silly now but - is there a difference between `**` and `__`? seems like both make it bold and afaik `**` is vastly more common.

Reply:
Same. I find the style `__` easier to read and I used it in other MR.